linux/dummy-web-app-src/Containerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.21
```

Comment:
Make `systemd` service instead for consistency -- we only had this in case we were gonna run it on the team servers.